validator/src/cli.rs

```diff
@@ -1194,6 +1194,12 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .help("Debug option to scan all append vecs and verify account index refcounts prior to clean")
                 .hidden(hidden_unless_forced())
         )
+        .arg(
+            Arg::with_name("accounts_db_test_skip_rewrites_but_include_in_bank_hash")
+                .long("accounts-db-test-skip-rewrites-but-include-in-bank-hash")
```

Comment:
my words here are so verbose. @brooksprumo or @HaoranYi do you have any better ideas?

Reply:
yes. updated.